src/main/java/io/github/heldev/shrinking/ShrinkingFacade.java

```diff
@@ -20,15 +20,13 @@ public ShrinkingFacade(Project project) {
 	}
 
 	public void enableShrinking() {
-		getEditorFactory().addEditorFactoryListener(NewEditorListener.get(project));
+		getEditorFactory().addEditorFactoryListener(NewEditorListener.get(project), project);
 
-		getProjectEditors()
-				.forEach(getService()::enableShrinking);
+		getProjectEditors().forEach(getService()::enableShrinking);
 	}
 
 	public void disableShrinking() {
-		getEditorFactory()
-				.removeEditorFactoryListener(NewEditorListener.get(project));
+		getEditorFactory().removeEditorFactoryListener(NewEditorListener.get(project));
```

Comment:
`removeEditorFactoryListener` is deprecated with no overload but I'm not a specialist in Idea plugins so I didn't remove it.